HelpSource/Classes/DrawGrid.schelp

```diff
@@ -138,6 +138,15 @@ A DrawGridY object that draws the y (vertical) axis
 returns:: a DrawGridY
 
 
+METHOD:: pixSpacing
```

Comment:
Update to `tickSpacing`?

Reply:
See my comments below about suggesting this be renamed to `minTickSpacing`

Reply:
@dyfer did your updated .schelp file not make it into your rebase?

Reply:
Hm, I don't think I included any helpfile changes, since it was outdated already. Help updates is still missing in this PR.